src/app/pages/news/9.4/10-project-migration-wizards.md

```diff
@@ -1,16 +1,24 @@
-## Project Migration Wizard {#migration}
+## Convert Project Wizard {#migration}
 
-User experience is a top priority in process automation projects. 
-That is why the Axon Ivy Portal has been massively enhanced.
+Converting an Axon Ivy project to the latest version and technologies was never easier. 
 
-- __Customizable Dashboard__: End-users can choose between different layouts, enable and disable default columns and even add custom columns at will.
-- __Default Widgets__: Axon Ivy Portal supports a sophisticated concept featuring default widgets for Process Starts, Task Lists, and Case Lists.
-- __Custom Widgets__: Individuality is king. End-users can easily create custom widgets in the dashboard.
-- __Adjustable Look for Process List__: Switch between an image, grid, and compact mode to display available processes.
+- __Convert Project Wizard__: The revised Convert Project Wizard converts Axon Ivy Projects 
+to the latest version. But now, it also helps to convert from Primefaces 7 to 11 and
+web service clients from Axis to CXF.
+- __Auto Conversion__: The Axon Ivy Engine auto converts all deployed projects to the
+latest version automatically.
```

Comment:
The Axon Ivy Engine auto converts all deployed projects automatically to the
latest version.